src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -397,13 +401,18 @@ object EventMessages {
                                            producer: Producer): SnsMessageDetails = {
     val topicArn = eventConfig("sns.topic.transform_engine_v2_in")
     val packageSignedUrl = generateS3SignedUrl(bucketName, s"$consignmentRef$tarExtension")
-    val packageShaSignedUrl = generateS3SignedUrl(bucketName, s"$consignmentRef$sh256256Extension")
-    val resource = Resource(value = packageSignedUrl)
-    val resourceValidation = ResourceValidation(value = packageShaSignedUrl)
-    val newBagit = BagitAvailable(resource, resourceValidation, consignmentRef)
-    val parameters = BagitAvailableParameters(newBagit)
-    val messageBody = TransferEngineV2InEvent(
-      `timestamp` = Timestamp.from(now).getTime, UUIDs = uuids, producer= producer, parameters = parameters).asJson.printWith(Printer.noSpaces)
+
+    val originator = "TDR"
+    val function = "tdr-export-process"
+    val consignmentType = producer.`type` match {
+      case "judgment" => ConsignmentType.JUDGEMENT
+      case _ => ConsignmentType.STANDARD
+    }
+
+    val properties = Properties(BagAvailable.getClass.getName, Timestamp.from(now).toString, function, TDR, UUID.randomUUID().toString, None)
```

Comment:
In the `Properties` model the last param `s3Key` is not the presigned url. It will actually be `{consignment reference}.tar.gz` for example `TDR-2023-24F.tar.gz`

There is a question for the TRE team around this. Do they require not just the bagit package, but also is "Sha256 tar" for the consignment as well. If so then the options are:

- TDR simply don't send the s3 Key value at all and TRE work out the s3 Keys based on the consignment reference
- Model is changed to send both s3 Keys
- TDR send two separate messages for both the bagit package and its Sha256 s3 object

Reply:
I've had a chat with Ian, and we've agreed to send the two s3 bucket keys as a json object:

```
...
{
  s3BagKey: xxxxxx
  s3Sha256Key: xxxxx
}
...
```

When you create the PR for correcting the spelling of the "judgment" you can add this change in as well.